engine/table/src/main/java/io/deephaven/engine/table/impl/QueryTable.java

```diff
@@ -638,7 +638,7 @@ public Table tail(final long size) {
     public Table headPct(final double percent) {
         if (percent < 0 || percent > 1) {
             throw new IllegalArgumentException(
-                    "headPct percentage provided should be between [0,1]: percent=" + percent);
+                    "headPct percentage of rows must be between [0,1]: : percent=" + percent);
```

Comment:
The Java parameter name is not needed and will be confusing for python users.
```suggestion
                    "percentage of rows must be between [0,1]: : percent=" + percent);
```